pod/live/pilotingInterface.py

```diff
@@ -56,21 +61,24 @@ def can_split(self) -> bool:
 
     @abstractmethod
     def check_piloting_conf(self) -> bool:
-        """Checks the piloting conf value."""
+        """Check the piloting conf value."""
         raise NotImplementedError
 
     @abstractmethod
     def is_available_to_record(self) -> bool:
-        """Checks if the broadcaster is available."""
+        """Check if the broadcaster is available."""
         raise NotImplementedError
 
     @abstractmethod
     def is_recording(self, with_file_check=False) -> bool:
         """
-        Returns if the broadcaster is recording state.
+        Return if the broadcaster is recording state.

```

Comment:
Et par conséquent mettre un truc du style : `Check if the broadcaster is recording state.`

Reply:
C'est fait ;)